packages/patternfly-4/react-core/src/components/Nav/examples/nav.scss

```diff
@@ -0,0 +1,5 @@
+.ws-preview {
+  & .pf-m-dark {
+    background-color: var(--pf-global--BackgroundColor--dark-300);
```

Comment:
can you set this on .ws-preview directly? would look better :)

Reply:
Setting it there directly changes it for all examples haha xD
I'm not sure if there's a better alternative to get the same effect. Tried to apply it on a wrapper div but it looks the same as the current setup.

Reply:
ah, then it's fine